tools/local-presubmit

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# A basic script to locally semi-simulate the build and test
+# operations performed by our continuous integration service. Note
+# that this does not do a clean and reinstall of npm modules and
+# sigh/bazel artifacts.
```

Comment:
Thanks, this file is going to be really helpful I think, especially for testing new Docker images.